.github/workflows/release-sims.yml

```diff
@@ -1,39 +0,0 @@
-name: Release Sims
-# Release Sims workflow runs long-lived (multi-seed & large block size) simulations
-# This workflow only runs on a pull request when the branch contains rc** (rc1/vX.X.x)
-on:
-  pull_request:
-    branches:
-      - "rc**"
-
-permissions:
-  contents: read
-
-jobs:
-  test-sim-multi-seed-long:
-    runs-on: buildjet-4vcpu-ubuntu-2004
-    if: "!contains(github.event.head_commit.message, 'skip-sims')"
-    timeout-minutes: 60
-    steps:
-      - uses: actions/checkout@v3
-      - uses: actions/cache@v3
-        with:
-          path: ~/go/bin
-          key: ${{ runner.os }}-go-runsim-binary
-      - name: Get data from Go build cache
-        # if: env.GIT_DIFF
-        if: ${{ false }}
-        uses: actions/cache@v3
-        with:
-          path: |
-            ~/go/pkg/mod
-            ~/.cache/golangci-lint
-            ~/.cache/go-build
-          key: ${{ runner.os }}-go-build-${{ hashFiles('**/go.sum') }}
-
-      - run: |
-          make build
-
-      - name: test-sim-multi-seed-long
-        run: |
-          make test-sim-multi-seed-long
```

Comment:
are we running this somewhere?

Reply:
Never run for 2 years, do we want to run it? The comment in the makefile does not lie, it is pretty long 😅

Reply:
I would say yes, can be its own nightly thing if needed

Reply:
Sure!